pkgs/development/compilers/dotnet/build-dotnet.nix

```diff
@@ -117,6 +122,24 @@ stdenv.mkDerivation rec {
 
     # Convert a "stdenv.hostPlatform.system" to a dotnet RID
     systemToDotnetRid = system: runtimeIdentifierMap.${system} or (throw "unsupported platform ${system}");
+
+    tests = {
+      version = testers.testVersion {
+        package = finalAttrs.finalPackage;
+      };
+      smoke-test = stdenv.mkDerivation {
```

Comment:
Maybe `runCommandNoCC` makes more sense? Looks a bit cleaner IMO.

Otherwise you should use `stdenvNoCC` to avoid pulling in the C compiler.

Reply:
good call, but `runCommandNoCC` seems to be an unused alias to `runCommand`, so I just used that instead